src/base/client.ts

```diff
@@ -30,7 +30,7 @@ export class ClientConfig {
   public baseUri?: string;
   public cacheManager?: ICacheManager;
   public headers?: BasicHeaders;
-  public parameters?: CommonParameters;
+  public parameters?: CommonParameters & CustomApiParameters;
```

Comment:
Pardon my typescript, but intersecting `CustomApiParameters` at this level to assign the type means that all API can essentially have "endpointName" etc defind in the param object, but it does nothing outside of the custom API helper. Is this correct?

Reply:
+1 to what Ben has said. Should this type be added to the `callCustomEndpoint` types in commerce-sdk?

Reply:
Let me take a look and see if there's a clean way to move this type to the commerce-sdk, it might require exposing a generic type for `ClientConfig`

Reply:
Done, updated the type to accept a generic type for parameters